src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/FaultyAnalysisGraph.java

```diff
@@ -0,0 +1,53 @@
+package com.amazon.opendistro.elasticsearch.performanceanalyzer.rca;
+
+import static com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.store.DummyGraph.DATA_NODE;
+import static com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.store.DummyGraph.LOCUS;
+
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.AnalysisGraph;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.Metric;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.Symptom;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.flow_units.SymptomFlowUnit;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.metrics.CPU_Utilization;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.metrics.Heap_AllocRate;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.metrics.Paging_MajfltRate;
+import com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.framework.api.metrics.Sched_Waittime;
+import java.util.Arrays;
+
+public class FaultyAnalysisGraph extends AnalysisGraph {
```

Comment:
What is the intent of this graph?

Reply:
This is for testing purpose so that we test the capture of failure metrics

Reply:
Can we move it into the tst folder?